Index.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-console */
+const express = require('express')
+
+const app = express()
+
+app.all('*', (request, response) => {
+  response.sendStatus(404)
+})
+
+app.listen(1337, () => {
```

Comment:
Nice. Now each one of you, when you pull this branch locally, should be able to start the express server, and type `http://localhost:1337` on your browser, and it should have the app running.

Reply:
nice job